source/documentation/ontology/__index.md

```diff
@@ -205,18 +204,40 @@ doesn't require a number to be the subject of an RDF statement. In
 OWL, this distinction is important: only object properties can
 be transitive or symmetric.
 
-The OWL language is sub-divided into three syntax classes:
-*OWL Lite*, *OWL DL* and *OWL Full*. OWL DL does not permit some
-constructions allowed in OWL Full, and OWL Lite has all the
-constraints of OWL DL plus some more. The intent for OWL Lite and
-OWL DL is to make the task of reasoning with expressions in that
-subset more tractable. Specifically, OWL DL is intended to be able
-to be processed efficiently by a
+The OWL language is sub-divided into several syntax classes:
+*OWL2 Full*, *OWL2 DL*, *OWL2 RL*, *OWL2 EL*, *OWL2 QL*,
+and also *OWL1 Lite*, *OWL1 DL* and *OWL1 Full*.
+The last three are deprecated now.
+OWL2 EL, OWL2 QL and OWL2 RL do not permit some constructions allowed in OWL2 Full and OWL2 DL.
+Although OWL1 is deprecated, Jena Ontology API is still support them.
```

Comment:
s/is still support them/still supports it (or is still supported)

Reply:
fixed